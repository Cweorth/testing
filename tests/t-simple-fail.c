#include <cut.h>

TEST(fail) {
    ASSERT(0);
}

