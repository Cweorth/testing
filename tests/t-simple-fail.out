b"[  1] fail.................................................................\x1b[1;31mFAIL\x1b[0m\n    assert '0' (simple-fail.c:4)\n\n\nSummary:\n  tests:       1\n  succeeded:   0\n  skipped:     0\n  failed:      1\n"